tests/components/vizio/test_config_flow.py

```diff
@@ -85,11 +106,13 @@ def vizio_cant_connect_fixture():
         yield
 
 
-async def test_user_flow_minimum_fields(hass: HomeAssistantType, vizio_connect) -> None:
+async def test_user_flow_minimum_fields(
+    hass: HomeAssistantType, vizio_connect, vizio_bypass_setup
+) -> None:
     """Test user config flow with minimum fields."""
     # test form shows
     result = await hass.config_entries.flow.async_init(
-        DOMAIN, context={"source": "user"}
+        DOMAIN, context={"source": SOURCE_USER}
```

Comment:
Please don't mix clean ups with fixes.

Reply:
should I revert this change?

Reply:
This time you can decide. But going forward please keep each PR to one purpose. Including random unrelated changes, clouds the PR and makes it easy to miss things that really matter.

If there are things to clean up, make a clean up PR where those things are addressed. That PR will probably be safe and easy to review.

Reply:
Understood. I will keep it here for now but note that for the future.